cluster-autoscaler/cloudprovider/hetzner/hack/update-vendor.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -o pipefail
+set -o nounset
+set -o errexit
+
+UPSTREAM_REPO=${UPSTREAM_REPO:-git@github.com:hetznercloud/hcloud-go.git}
```

Comment:
To confirm, but this won't work if the user of the script doesn't have an ssh key configured to clone from Github.

Maybe you should use the https URL to clone the repo.

Reply:
Updated :+1: